testsuite/constant_pH.py

```diff
@@ -64,14 +64,14 @@ def setUpClass(cls):
                                 cls.system.box_l, type=cls.type_H)
 
         cls.RE.add_reaction(
-            Gamma=cls.K_HA_diss_apparent, reactant_types=[
+            Gamma=cls.Ka, reactant_types=[
                 cls.type_HA], reactant_coefficients=[1], product_types=[
                 cls.type_A, cls.type_H], product_coefficients=[
                 1, 1], default_charges={cls.type_HA: 0, cls.type_A: -1, cls.type_H: +1})
         cls.RE.constant_pH = cls.pH
 
     @classmethod
-    def ideal_degree_of_association(cls, pH):
+    def ideal_alpha(cls, pH):
```

Comment:
this is the formula for the degree of association= 1-degree of dissociation=1-alpha

Reply:
Looks to me like you get away with exchanging the degree of association and the degree of dissociation because both are 0.5 here. Therefore I would choose a point in the parameter space where both are distinct.